interface/src/app/analyze/analyze.js

```diff
@@ -31,9 +31,19 @@ angular.module('adage.analyze', [
   $anchorScroll.yOffset = 80;
 }])
 
-.controller('AnalyzeCtrl', ['$scope', '$log', '$location',
+.controller('AnalyzeCtrl', ['$scope', '$stateParams', '$log', '$location',
   '$anchorScroll', 'Sample',
-  function AnalyzeCtrl($scope, $log, $location, $anchorScroll, Sample) {
+  function AnalyzeCtrl($scope, $stateParams, $log, $location, $anchorScroll,
+                       Sample) {
+    $scope.isValidModel = false;
+    // Do nothing if mlmodel in URL is falsey. The error will be taken
+    // care of by "<ml-model-validator>" component.
+    if (!$stateParams.mlmodel) {
+      return;
```

Comment:
Rats. I'll need to try again to get your fork to build. If I can trace into the code I can see what's happening.